next.config.js

```diff
@@ -947,6 +947,14 @@ module.exports = withFonts(
       FATHOM_ID: 'FOEMPXUV',
     },
     redirects,
+    async rewrites() {
+      return [
+        {
+          source: '/',
+          destination: '/index',
```

Comment:
It doesn't work on the Vercel deployment on the main page, but it works if you click on the home link at the left... 
Can you try adding the /. So changing `          destination: '/index',` to `           destination: '/index/',`